entrypoint.sh

```diff
@@ -33,4 +33,4 @@ else
     echo "No extra plugin packages provided. Specify 'OTEAPI_PLUGIN_PACKAGES' to specify plugin packages."
 fi
 
-hypercorn asgi:app --bind 0.0.0.0:8080 "$@"
+python -m debugpy --wait-for-client --listen 0.0.0.0:5678 -m hypercorn --bind 0.0.0.0:8080 --reload asgi:app "$@"
```

Comment:
Running it like this should only happen for the development target. There should be some form of an `if/else` block here.